src/SprykerEco/Zed/AvalaraTax/Business/Executor/AvalaraTransactionExecutor.php

```diff
@@ -78,6 +84,10 @@ public function executeAvalaraCreateTransaction(
         CalculableObjectTransfer $calculableObjectTransfer,
         string $transactionTypeId
     ): AvalaraCreateTransactionResponseTransfer {
+        if (static::$avalaraCreateTransactionResponseCache) {
```

Comment:
* The cache should also take into account changes in the `CalculableObjectTransfer`.

Reply:
Resolved.